gazelle/internal/swift/is_builtin_module.go

```diff
@@ -2,30 +2,48 @@ package swift
 
 import mapset "github.com/deckarep/golang-set/v2"
 
-var otherBuiltinModules = mapset.NewSet[string](
-	"RegexBuilder", // TODO: not included in System/Library/Frameworks for some reason
+var builtinTestFrameworks = mapset.NewSet[string](
 	"XCTest",
 )
 
 // The list of frameworks is found in builtin_modules.go.
-var allBuiltInModuleSets = []mapset.Set[string]{
-	otherBuiltinModules,
+var allBuiltInFramewrokSets = []mapset.Set[string]{
```

Comment:
Can you fix the spelling? I think that it should be `allBuiltInFrameworkSets`.

Reply:
Oops yes